internal/blocktx/processor_test.go

```diff
@@ -473,7 +472,7 @@ func TestStartFillGaps(t *testing.T) {
 			logger := slog.New(slog.NewTextHandler(os.Stdout, &slog.HandlerOptions{Level: slog.LevelInfo}))
 
 			blockRequestCh := make(chan blocktx.BlockRequest, 10)
-			var blockProcessCh chan *p2p.BlockMessage = nil
+			var blockProcessCh chan *p2p.BlockMessage
```

Comment:
Not a major, but this nil assignment was purposeful, to show that this will not be used in the test.

Personally, I would like this to stay with `= nil`

Reply:
I think with `//nolint` it should be possible to allow the exception here.

Reply:
Revert this changes and added `// nolint: revive` comment